designsafe/apps/data/views/base.py

```diff
@@ -207,7 +209,9 @@ def get_context_data(self, **kwargs):
         } for user in getattr(pub, 'users', [])]
         context['publication'] = pub
         context['description'] = pub.project.value.description
-        
+        context['experiments'] = getattr(pub, 'experimentsList', [])
+        context['missions'] = getattr(pub, 'missions', [])
+        context['simulations'] = getattr(pub, 'simulations', [])
         if self.request.user.is_authenticated:
```

Comment:
I think you might need to add `hybridSimulations` in here as well. (we don't have any published at the moment, but the model exists in our system.)

Reply:
Added in a new commit. I used hybrid_simulation.py in apps/projects/models/agave as a reference.